Sources/Sentry/SentryReplayEvent.m

```diff
@@ -0,0 +1,28 @@
+#import "SentryReplayEvent.h"
+#import "SentryDateUtil.h"
+#import "SentryId.h"
+
+@implementation SentryReplayEvent
+
+- (NSDictionary *)serialize
+{
+    NSMutableDictionary *result = [[super serialize] mutableCopy];
+
+    NSMutableArray *trace_ids = [NSMutableArray array];
+
+    for (SentryId *traceId in self.traceIds) {
+        [trace_ids addObject:traceId.sentryIdString];
+    }
+
+    result[@"urls"] = self.urls;
+    result[@"replay_start_timestamp"] =
+        @([SentryDateUtil javascriptDate:self.replayStartTimestamp]);
+    result[@"trace_ids"] = trace_ids;
+    result[@"replay_id"] = self.replayId.sentryIdString;
+    result[@"segment_id"] = @(self.segmentId);
+    result[@"replay_type"] = @"buffer";
```

Comment:
missing a `// TODO:` here? In JS there are two values;

https://github.com/getsentry/sentry-javascript/blob/bc79d32932d8d8a96f5238989a5d9ab3db915164/packages/types/src/replay.ts#L27

We send to Sentry either value to indicate whether the session started due to session sampling, or if it was buffering (keeping 30 seconds in memory 'in case' there's an error, or another trigger to start capturing the recording)

<img width="662" alt="image" src="https://github.com/getsentry/sentry-cocoa/assets/1633368/451bc45e-a3a9-46c4-b5c2-d269c613b869">
[Session example](https://sentry.sentry.io/replays/db7635c5e8b34421887ecc606fb6f186/?project=11276&query=&referrer=%2Freplays%2F&statsPeriod=24h&t_main=tags&yAxis=count%28%29)

[buffer](https://sentry.sentry.io/replays/9320fe38194f47629b028e11d77358dd/?project=11276&query=&referrer=%2Freplays%2F&statsPeriod=24h&t_main=tags&yAxis=count%28%29) example:
<img width="647" alt="image" src="https://github.com/getsentry/sentry-cocoa/assets/1633368/2e816010-ef38-4102-97ba-51ef78f03651">